pkgs/top-level/all-packages.nix

```diff
@@ -18482,6 +18482,7 @@ with pkgs;
   libffcall = callPackage ../development/libraries/libffcall { };
 
   libffi = callPackage ../development/libraries/libffi { };
+  libffi7 = callPackage ../development/libraries/libffi/7.nix { };
```

Comment:
maybe just call it libffi3_3?

Reply:
or libffi_so7

Reply:
I'll go with libffi_3_3 to match most other things in nixpkgs (e.g. terraform_0_15).

Reply:
SGTM